policy/acceptable-use.md

```diff
@@ -0,0 +1,32 @@
+## Terminology
+
+2i2c Managed Infrastructure: Any web service, program, or cloud infrastructure that 2i2c manages on behalf of a user’s community.
+
+## Acceptable Use Policy
+
+This policy does not nullify any part of the Acceptable Use Policies that apply to your community (e.g., university Acceptable Use Policies).
+In addition you must comply with the policies and guidelines for any specific set of resources to which you have been granted access (e.g., policies from Cloud Providers).
+When other policies are more restrictive than this policy, the more restrictive policy takes precedence.
+
+- You may use only the 2i2c Managed Infrastructure and accounts for which you have authorization.
+- You may not use another individual's account, or attempt to capture or guess other users' passwords.
+  You may not enable unauthorized users to access infrastructure managed by 2i2c by providing them access to your account.
+- You are individually responsible for appropriate use of all resources provided to you, including your interactive computing session, networking infrastructure on your hub, and the software and hardware you use.
+- 2i2c is bound by its contractual and license agreements respecting certain third party resources; you are expected to comply with all such agreements when using such resources.
+- You should make a reasonable effort to protect your passwords and to secure resources against unauthorized use or access.
+  Where applicable you must configure access to other cloud services or data in a way that reasonably prevents unauthorized users from accessing them.
+- You must not attempt to access restricted portions of the network or any 2i2c Managed Infrastructure without appropriate authorization by a 2i2c engineer.
+- You must not use 2i2c Managed Infrastructure and/or network resources in conjunction with the execution of programs, software, processes, or automated transaction-based commands that are intended to disrupt (or that could reasonably be expected to disrupt) other computer or network users, or damage or degrade performance, software or hardware components of a system.
+- Do not use the 2i2c Managed Infrastructure to distribute or facilitate the sending of unsolicited or unlawful (i) email or other messages, or (ii) promotions of any kind;
+- Do not use the 2i2c Managed Infrastructure to engage in or promote any other fraudulent, deceptive or illegal activities.
+- Do not use the 2i2c Managed Infrastructure to process, store or transmit material, including any Customer Data, in violation of any Law or any third party rights, including without limitation privacy rights;
+- Do not use the 2i2c Managed Infrastructure in any circumstances where failure could lead to death, personal injury or environmental damage, and you further acknowledge that the 2i2c Managed Infrastructure is not designed or intended for such use.
```

Comment:
I would like us to explicitly mention crypto-mining as an activity that contravenes our acceptable use policy. I think we allude to it with "automated transaction-based commands" - but I would like to be crystal clear that we will not look kindly upon users found crypto-mining